src/lib/store/remote/helpers.js

```diff
@@ -106,13 +108,21 @@ exports.parseFromRemote = function(state, object) {
   // as in some cases IDs might contain '/', too
   //
   matches = id.match(/([^\/]+)\/(.*)/);
+  if (matches == null) {
+    console.log('hoodie: unhandled id ' + id);
```

Comment:
we can't do `console.log`s, it breaks some browsers. Then it's good to merge, I'll add a test for it in https://github.com/hoodiehq/hoodie-integration-test/. Thanks for your help Thomas!